src/views/RequiredSignoffs/ViewSignoff/index.jsx

```diff
@@ -165,8 +165,23 @@ function ViewSignoff({ isNewSignoff, ...props }) {
     }
   };
 
-  // TODO: Add delete logic
-  const handleSignoffDelete = () => {};
+  const handleSignoffDelete = async () => {
+    const { error } = await saveRS({
+      product: productTextValue,
+      channel: channelTextValue,
+      // pass nothing for current and additional roles
+      // to make sure all existing roles are scheduled
+      // for deletion.
+      roles: [],
+      originalRoles,
+      additionalRoles: [],
+      isNewSignoff,
+    });
+
+    if (!error) {
+      props.history.push('/required-signoffs');
+    }
+  };
```

Comment:
Any chance we could have the delete preceded with a confirmation dialog?

Reply:
Yeah, that's probably a good idea given that it's a pretty significant event. Will do.